agreements/create_lots.sql

```diff
@@ -0,0 +1,201 @@
+ 1,
+ 'Products',
+ '31-Oct-2016',
+ '31-Mar-2020');
+ 
+ INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(4,
+ 'Lot 4',
+ 'Information Assured Products',
+ 'Information Assured Products description',
+ 1,
+ 'Products',
+ '31-Oct-2016',
+ '31-Mar-2020');
+ 
+ INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(5,
+ 'Lot 5',
+ 'Volume Hardware Requirements (Direct from Original Equipment Manufacturer (OEM))',
+ 'Volume Hardware Requirements (Direct from Original Equipment Manufacturer (OEM))',
+ 1,
+ 'Products',
+ '31-Oct-2016',
+ '31-Mar-2020');
+ 
+ INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(6,
+ 'Lot 6',
+ 'Catalogue',
+ 'TCatalogue description',
+ 1,
+ 'Products',
+ '31-Oct-2016',
+ '31-Oct-2021');
+ 
+ -- RM6068 CA FK 2, 4 Lots
+ 
+ INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(7,
+ 'Lot 1',
+ 'Hardware & Software & Associated Services',
+ 'Hardware & Software & Associated Services description',
+ 2,
+ 'Services',
+ '10-Dec-2019',
+ '09-Dec-2021');
+ 
+ INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(8,
+ 'Lot 2',
+ 'Hardware and Associated Services',
+ 'Hardware and Associated Services description',
+  2,
+ 'Services',
+ '10-Dec-2019',
+ '19-Dec-2021');
+ 
+ INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(9,
+ 'Lot 3',
+ 'Software and Associated Services',
+ 'Software and Associated Services description',
+  2,
+ 'Services',
+ '10-Dec-2019',
+ '09-Dec-2021');
+ 
+ INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(4,
```

Comment:
Duplicate key - id 4 already exists (I will change to `10` locally to continue testing)